.github/workflows/workflow-httpstan.yml

```diff
@@ -0,0 +1,341 @@
+          done
+      - name: Test wheel
+        env:
+          CC: ${{ matrix.compiler }}
+        if: success()
+        run: |
+          cd dist
+          python -m venv test_env --copies --clear
+          ./test_env/Scripts/activate
+          which python
+          python -m pip install --upgrade pip
+          for whl in *.whl; do break; done
+          python -m pip install "${whl}"
+          python -m pip install pytest
+          python -m pip freeze
+          python -m pytest -s -v --cov=httpstan --cov-fail-under=93 --forked ../tests
+          cd ..
+      - uses: actions/upload-artifact@v2
+        if: success()
+        with:
+          name: httpstan_wheels
+          path: dist/*.whl
+      - name: Upload wheel to release
+        uses: svenstaro/upload-release-action@v1-release
+        if: success() && startsWith(github.ref, 'refs/tags/')
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          file: dist/*.whl
+          tag: ${{ github.ref }}
+          overwrite: true
+          file_glob: true
+      - name: Upload to pypi
+        if: success() && startsWith(github.ref, 'refs/tags/')
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_PASSWORD }}
+        run: |
+          python -m pip install --no-cache-dir twine
+          twine upload --repository-url https://test.pypi.org/legacy/ -u ${TWINE_USERNAME} -p ${TWINE_PASSWORD} --skip-existing dist/*
+
+  #### WHEEL (macOS) ####
+  publish_macos:
+    name: macos wheel
+    needs: [pytest, tox-flake8, tox-format, tox-mypy, tox-docs]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        python-version: [3.7, 3.8]
+        compiler: [clang]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: check C++ compiler and Python
+        run: |
+          clang --version
+          clang++ --version
+          which clang
+          python --version
+          which python
+      - name: Install dependencies
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          make
+      - name: Build wheel
+        env:
+          CC: ${{ matrix.compiler }}
+        run: |
+          python -m poetry run pip install --no-cache-dir Cython
+          python -m poetry build -v
+      - name: Test wheel
+        env:
+          CC: ${{ matrix.compiler }}
+        if: success()
+        run: |
+          cd dist
+          python -m venv test_env --copies --clear
```

Comment:
Are --copies and --clear necessary?

Reply:
Probably not, I copied this from my other repo